docs/writing-tests/snapshot-testing.md

```diff
@@ -110,8 +110,9 @@ Visual tests capture images of stories and compare them against image baselines.
 
 #### Learn about other UI tests
 
+- [Test runner](./test-runner.md) to automate test execution
 - [Visual tests](./visual-testing.md) for appearance
 - [Accessibility tests](./accessibility-testing.md) for accessibility
 - [Interaction tests](./interaction-testing.md) for user behavior simulation
 - Snapshot tests for rendering errors and warnings
-- [Import stories in other tests](./importing-stories-in-tests.md) for other tools
+- [Import stories in other tests](./importing-stories-in-tests.md) for other tools
```

Comment:
apparently I can't add comments to images so I'm just adding a comment to the piece that is close to the image in the PR review page. 

I feel like [the images](https://github.com/storybookjs/storybook/blob/29bc7a43100fa19b12394b3d0d99398a4451f472/docs/writing-tests/storybook-interaction-test-runner-loginform-optimized.png) could be cropped so they get scaled and the text becomes easier to read. WDYT?